scripts/characters/constructor.gd

```diff
@@ -137,7 +137,7 @@ func newbaby(mother,father):
 	#Prep
 	person.race = ''
 	person.age = ''
-	person.sex = ''
+	#person.sex = ''
```

Comment:
When does person.sex actually ever get assigned? It looks like the commented code down below was supposed to do it, but you weren't the one to comment that out. From what it looks like to me, children are just gonna come out with null genders.

Reply:
nvm you're good Im dumb